flexmeasures/cli/data_edit.py

```diff
@@ -78,6 +79,20 @@ def fm_edit_data():
     type=int,
     help="Set the attribute to this integer value.",
 )
+@click.option(
+    "--list",
+    "attribute_list_value",
+    required=False,
+    type=str,
+    help="Set the attribute to this list value. Pass a string with a JSON-parse-able list representation, e.g. '[1,\"a\"]'.",
+)
+@click.option(
+    "--dict",
+    "attribute_dict_value",
+    required=False,
+    type=str,
+    help="Set the attribute to this dict value. Pass a string with a JSON-parse-able dict representation, e.g. '{1:\"a\"}'.",
+)
```

Comment:
how dictionary is going to help us in editing the attributes. Let's consider, I want to show [1,[2,3]] sensors for asset --id 1. How this could be written in a dictionary?

Reply:
Well, for that attribute it doesn't make sense, but other attributes might require dicts.
For instance, we plan to store the flex-model on the asset.

Reply:
it should be mentioned somewhere, to tell the users to use dicts and lists as required.

Reply:
We haven't documented attributes well, so far. 
That is also because just a few of them are expected, and developers are free to make their own for custom functionality.

But the few we work with should have a place, that's true.
Not necessary in this PR, though